src/modules/reserve/countreserve.js

```diff
@@ -0,0 +1,11 @@
+import getReserverations from './getReserverations.js';
+
+const countReserve = async (id) => {
```

Comment:
Great job team🤝

- [ ] As the previous reviewers suggested, kindly add a reservations counter on your popup and make sure this reservations counter is updated every time a new reservation is submitted.

Reply:
* Kindly add tests to cover all your count functions as its a requirement for your capstone project 
![image](https://user-images.githubusercontent.com/52497246/221392137-16272ada-7426-41e4-8a8e-5ab1af2bf275.png)